parts/kubernetesmastercustomscript.sh

```diff
@@ -130,6 +130,23 @@ function ensureKubelet() {
 function extractKubectl(){
     systemctl enable kubectl-extract
     systemctl restart kubectl-extract
+    kubectlExtracted=1
```

Comment:
Yes, I was thinking something similar. I still think the service should be restartable so that systemd restarts it in the case that it does happen to fail. And then we can remove the retry code you just recently added? Yeah? If that works out, I think it'd be okay.